applications/StructuralMechanicsApplication/tests/test_solid_2p5d_element.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+# Importing analysis
+from KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_analysis import StructuralMechanicsAnalysis
+
+# Importing post-process
+from KratosMultiphysics.vtk_output_process import VtkOutputProcess
+from KratosMultiphysics.gid_output_process import GiDOutputProcess
+from KratosMultiphysics.json_output_process import JsonOutputProcess
+from KratosMultiphysics.from_json_check_result_process import FromJsonCheckResultProcess
+
+class TestSolid2p5DElement(KratosUnittest.TestCase):
```

Comment:
I would prefer since this is a standard test (i.e. has ProjectParameters etc) to add it in the standard way (i.e. in the test-factory)